### ISSUE TITLE
Remove .env support

### ISSUE THREAD

**Issue by haykam821** [haykam821/ShortcutsPreview#5]
Support for `.env` has been deprecated since 37a436a955c8c165a617d05806cc463ebf7af227. It works with the new config system, although specifying something in `config.json` overrides it.

**Comment by Hacksore** [comment]
Not sure if this is a separate issue but when running my instance I could not get the bot to run reddit without the following additional config item.

This might just be a documentation issue? 

Added the `refreshToken`
```json
{
    "global": {
        "enabled": true
    },
    "reddit": {
        "credentials": {
            "clientId": "xD",
            "clientSecret": "xD",
            "refreshToken": "xD"
        }
    },
    "discord": {
        "token": "xD"
    }
}
```

**Comment by haykam821** [comment]
@Hacksore Yes, that is just bad documentation. I have made a pull request, #8, to enhance the documentation for the config.

**Comment by Hacksore** [comment]
@haykam821 can we please close this?
